lib/std/debug.zig

```diff
@@ -882,7 +888,11 @@ fn printUnknownSource(debug_info: *DebugInfo, out_stream: anytype, address: usiz
 pub fn printUnwindError(debug_info: *DebugInfo, out_stream: anytype, address: usize, err: UnwindError, tty_config: io.tty.Config) !void {
     const module_name = debug_info.getModuleNameForAddress(address) orelse "???";
     try tty_config.setColor(out_stream, .dim);
-    try out_stream.print("Unwind information for `{s}:0x{x}` was not available ({}), trace may be incomplete\n\n", .{ module_name, address, err });
+    if (err == error.MissingDebugInfo) {
```

Comment:
`UnwindError` reads like an error set:
```
pub const UnwindError = if (have_ucontext)
    @typeInfo(@typeInfo(@TypeOf(StackIterator.next_unwind)).Fn.return_type.?).ErrorUnion.error_set
else
    void;
```
No idea, why the output gives the error union though:
```
    pub const ErrorUnion = struct {
        error_set: type,
        payload: type,
    };
```
```
/home/ci/actions-runner10/_work/zig/zig/lib/std/debug.zig:891:13: error: operator == not allowed for type 'error{MissingDebugInfo}!void'
/home/ci/actions-runner10/_work/zig/zig/lib/std/debug.zig:891:13: error: operator == not allowed for type 'error{MissingDebugInfo}!void'
```

Reply:
I think this error is because on the platforms where it's failing, it can only be one value - so the comparison isn't valid. Working on a fix.